lib/jellyfish/component.ex

```diff
@@ -133,7 +134,17 @@ defmodule Jellyfish.Component do
          properties: properties
        }}
     else
-      {:error, _reason} = error -> error
+      false ->
+        raise("""
+          #{inspect(type)} can only be used if JF_#{type |> to_string!() |> String.upcase()}_USED environmental variable is set to \"true\"
+        """)
```

Comment:
This implementation was copied over. However, I'm not sure whether it'd be better to raise here or return an error tuple

Reply:
I think we should return error tuple here. The motivation is that we can have the same backend with differently configured cloud instances, or the configuration could change for some reason. We don't want to crash the Jellyfish if such thing happens.
You may also change this behavior from wherever you copied it from - if you think it's appropriate.

Reply:
Changed in WebRTC as well